x-pack/test/search_sessions_integration/tests/apps/discover/async_search.ts

```diff
@@ -25,15 +25,23 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   ]);
   const searchSessions = getService('searchSessions');
   const retry = getService('retry');
+  const kibanaServer = getService('kibanaServer');
 
   describe('discover async search', () => {
     before(async () => {
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
-      await esArchiver.load('x-pack/test/functional/es_archives/discover/default');
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/discover/default'
+      );
       await PageObjects.common.navigateToApp('discover');
       await PageObjects.timePicker.setDefaultAbsoluteRange();
       await PageObjects.header.waitUntilLoadingHasFinished();
     });
+    after(async () => {
+      await kibanaServer.importExport.load(
```

Comment:
Was the intention to `unload`?

Reply:
Darnit!  Bad regex!   Fixing up now.  Thanks Anton!